### ISSUE TITLE
MIS: Input Verification Model read_geoinfo()

### ISSUE THREAD

**Issue by omltcat** [CynthiaLiu0805/BridgeCorrosion#48]
Do you call read_geoinfo() once during initialization, or every time when you need to verify a coordinate?

I hope it is the first case, as you don't want to read the file every time for every request. But if so, a state variable is needed to store the shape of Ontario in memory. 

Another thing regarding the execption:
> filename == ∅

Means the filename string is empty. I think what you meant to say here is the file itself cannot be found.

**Comment by CynthiaLiu0805** [comment]
You are correct, I was mixing up those math notations. Those are fixed in 66f0556.
